conf/machine/freedom-u540-common.inc

```diff
@@ -18,16 +19,10 @@ QEMUVERSION = "riscv"
 
 EXTRA_IMAGEDEPENDS += "riscv-pk"
 
-SERIAL_CONSOLE = "115200 ttyS0"
+SERIAL_CONSOLE = "115200 console"
 
 MACHINE_EXTRA_RRECOMMENDS += " kernel-modules"
 
-RISCV_BBL_PAYLOAD ?= "${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin"
-
-INITRAMFS_IMAGE_BUNDLE = "1"
-INITRAMFS_IMAGE = "riscv-initramfs-image"
-KERNEL_INITRAMFS = '-initramfs'
-
```

Comment:
Just comment above three vars out and add a comment asking to enable them if someone wants initramfs image.

Reply:
Agreed on this. No need for two machines. Could convert above to a "config" as well, but for now I feel it is OK to comment it out with a comment what it means to un-comment it.